.github/ISSUE_TEMPLATE/01_bug_report.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug report
+description: Create a bug report
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for reporting a bug! Please note:
+        - For SWC plugin API compatibility issues, check the [compatibility table](https://github.com/lingui/swc-plugin#compatibility).
+        - Bug reports without reproduction steps may be closed.
```

Comment:
I believe people may not realize that problems reported by the nextjs/swc is due to compatibility, i would rather write here excplitly something like: 

if you see "Panic! bla bla" that means you use incompatible version of SWC plugin to th tuntime version check this issue for more info